recipes-devtools/gie/tensorrt-samples_8.0.1-1.bb

```diff
@@ -0,0 +1,69 @@
+DESCRIPTION = "NVIDIA TensorRT Samples for deep learning"
+LICENSE = "Proprietary"
+
+inherit l4t_deb_pkgfeed cuda
+
+HOMEPAGE = "http://developer.nvidia.com/tensorrt"
+
+L4T_DEB_GROUP = "tensorrt"
+
+SRC_COMMON_DEBS = "\
+    libnvinfer-samples_${PV}+cuda10.2_all.deb;downloadfilename=libnvinfer-samples_${PV}+cuda10.2_all.deb;name=samples;subdir=tensorrt \
+"
+
+SRC_URI:append = " file://0001-Makefile-fix-cross-compilation-issues.patch"
```

Comment:
The `:` separator is supported in dunfell, but I'd prefer to keep things aligned with the rest of the branch and use the old `_` separator.

Reply:
Yes, I know. My bad I forget that one 
I will fix it